integration-tests/narayana-stm/src/main/resources/jbossts-properties.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE properties SYSTEM "http://java.sun.com/dtd/properties.dtd">
+
+<properties>
+    <entry key="ObjectStoreEnvironmentBean.objectStoreDir">/tmp/data/tx-object-store</entry>
+    <entry key="com.arjuna.ats.arjuna.common.ObjectStoreEnvironmentBean.communicationStore.objectStoreDir">/tmp/data/tx-object-store</entry>
+    <entry key="com.arjuna.ats.arjuna.common.ObjectStoreEnvironmentBean.stateStore.objectStoreDir">/tmp/data/tx-object-store</entry>
```

Comment:
Would it be possible to make them Quarkus config (i.e. in the `application.properties`)?

Reply:
I removed the properties file (it is for when we run on OpenShift)

Reply:
Well, if they are useful, I think in the long run we should expose them as Quarkus config.

OpenShift is our environment of choice and things should be easily configurable.

Not saying it should block this PR though but it would be useful to plan that for the future.